javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var newArr = [];
+  for (let i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This isn't filtering out strings, it's keeping numbers. What if there were other values in the array such as booleans or objects?

Reply:
Hey Cody, Thanks for the feedback! 

I commented some pseudocode in the filter-out-strings.js 
Let me know what you think!